osgp/shared/osgp-kafka-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/kafka/da/signature/MessageSigner.java

```diff
@@ -56,8 +57,13 @@
     public static final String DEFAULT_SIGNATURE_KEY_ALGORITHM = "RSA";
     public static final int DEFAULT_SIGNATURE_KEY_SIZE = 2048;
 
+    // Two magic bytes (0xC3, 0x01) followed by an 8-byte fingerprint
+    public static final int AVRO_HEADER_LENGTH = 10;
+
     private boolean isSigningEnabled;
 
+    private boolean isStripHeaders;
```

Comment:
I thought that our convention was to use the `is` prefix only for getters, not for fields?